src/javaRosa.js

```diff
@@ -951,20 +948,18 @@ define([
                 widget.setValue(defaultValue);
                 widget.handleChange();
             } else {
-                var itextItem = widget.getItextItem();
+                var value = widget.getItextValue();
 
-                if (!itextItem) {
-                    widget.setValue("");
-                    return;
+                if (!_.isString(value)) {
```

Comment:
I did some more digging. In a test case like this: https://github.com/dimagi/Vellum/blob/bec373ab8ac1ceb586ed6c05a377a11289de8ead/tests/static/javaRosa/test-xml-1.xml#L25-L34 there isn't an itext for question1 so this will create the itext for hin with the same value as en. This could probably use some sort of refactor, but I think it may be out of scope for this particular fix.